examples/awssns-quickstart/src/main/java/awssns/Main.java

```diff
@@ -0,0 +1,13 @@
+package awssns;
+
+import javax.enterprise.inject.se.SeContainer;
+import javax.enterprise.inject.se.SeContainerInitializer;
+
+public class Main {
+
+    public static void main(String[] args) {
+    	
+        SeContainer container = SeContainerInitializer.newInstance().initialize();
+        container.getBeanManager().createInstance().select(BeanUsingAnEmitter.class).get().periodicallySendMessage();
```

Comment:
it is the GitHub view, or is there an indentation issue.

Reply:
it should be only one empty line. I removed it just in case :).